tools/generators/ethereum/contract_parsing.go

```diff
@@ -72,10 +72,12 @@ type contractInfo struct {
 	Events           []eventInfo
 }
 
-type paramInfo struct {
-	Name      string
-	Type      string
-	ParsingFn string
+type argInfo struct {
```

Comment:
What do you think about naming this struct `cmdArgInfo`?

Reply:
c98f1139e3f3d2b679388ec0817916fa31e6cc5d